index.html

```diff
@@ -152,38 +164,41 @@ <h2 class="cover__subtitle">LET’S CONNECT</h2>
         <a href="#" class="about-me__text-button">Get my resume</a>
       </div>
       <div class="about-me__skills">
-        <ul class="about-me__skills-items">
-          <li class="about-me__skills-language">
+        <div class="about-me__skills-items">
+          <div class="about-me__skills-language">
             <h3 class="about-me__skills-text">Languages</h3>
             <img class="about-me__deploy" src="./images/deploy.png" alt="deploy symbol">
-          </li>
-          <li class="about-me__skills-tag">
+          </div>
+          <div class="about-me__languages">
+            <div class="about-me__skills-tag">
               <img src="./icons/javascript.svg" alt="javascript">
               <span>JavaScript</span>
-          </li>
-          <li class="about-me__skills-tag">
-            <img class="sixht-sheet__developer-language-img" src="./icons/html.svg" alt="html">
-            <span>HTML</span>
-          </li>
-            <li class="about-me__skills-tag">
+            </div>
+            <div class="about-me__skills-tag">
+              <img class="sixht-sheet__developer-language-img" src="./icons/html.svg" alt="html">
+              <span>HTML</span>
+            </div>
+            <div class="about-me__skills-tag">
               <img class="sixht-sheet__developer-language-img" src="./icons/css.svg" alt="css">
               <span>CSS</span>
-          </li>
-          <li class="about-me__skills-language">
+            </div>
+          </div>
+          <div class="about-me__skills-language">
             <h3 class="about-me__skills-text">Frameworks</h3>
             <img class="about-me__deploy" src="./images/deploy-right.png" alt="deploy right">
-          </li>
-          <li class="about-me__skills-language">
+          </div>
+          <div class="about-me__skills-language">
             <h3 class="about-me__skills-text">Skills</h3>
             <img class="about-me__deploy" src="./images/deploy-right.png" alt="deploy rigth">
-          </li>
-        </ul>
+          </div>
+        </div>
       </div>
     </section>
     <section class="formulary">
       <h2 class="formulary__title">Contact me</h2>
       <p class="formulary__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-      <img class="formulary__image" src="./images/form-shapes-mobile@2x.svg" alt="backgroung form">
+      <img class="formulary__image1" src="./images/form-shapes-mobile@2x.svg" alt="backgroung form">
+      <img class="formulary__image2" src="./images/contact-form-shapes-desktop.svg" alt="backgroung form">
       <form action="https://formspree.io/f/xeqnpkpe" method="post">
```

Comment:
- [ ] [REQUIRED] Please, make sure that the form matches the figma design. In your website the form is aligned to the left. All you need to do is to put it in center.

![image](https://user-images.githubusercontent.com/60979926/177663935-aa81bf34-1153-4cd2-a9bf-dfa8631d10c7.png)